tests/test_client.py

```diff
@@ -23,8 +24,10 @@
 from vt import Client
 from vt import Object
 
+from tests import wsgi_app
 
-def new_client(httpserver):
+
+def new_client(httpserver, unused_apikey=""):
```

Comment:
```suggestion
def new_client(httpserver):
```

Reply:
That API key is required so this monkeypatch I made works as expected:
https://github.com/VirusTotal/vt-py/blob/7c22eca44f5458dbee097141884410c6936bf5b0/tests/test_client.py#L503-L505